sssom/util.py

```diff
@@ -727,61 +727,40 @@ def deal_with_negation(df: pd.DataFrame) -> pd.DataFrame:
             "The dataframe, after assigning default confidence, appears empty (deal_with_negation)"
         )
 
-    #  If s,!p,o and s,p,o , then prefer higher confidence and remove the other.  ###
-    negation_df: pd.DataFrame
-    negation_df = df.loc[df[PREDICATE_MODIFIER] == PREDICATE_MODIFIER_NOT]
-    normalized_negation_df = negation_df.reset_index()
-
-    # This step ONLY if 'NOT' is expressed by the symbol '!' in 'predicate_id' #####
-    # normalized_negation_df[PREDICATE_ID] = normalized_negation_df[
-    #     PREDICATE_ID
-    # ].str.replace("!", "")
-    ########################################################
-    normalized_negation_df = normalized_negation_df.drop(["index"], axis=1)
-
-    # remove the NOT rows from the main DataFrame
-    condition = negation_df.isin(df)
-    positive_df = df.drop(condition.index)
-    positive_df = positive_df.reset_index().drop(["index"], axis=1)
-
```

Comment:
I am very worried about this. can you describe the concrete consequences of this change? I didn't mean to prompt a redesign in this case, only to say that there very few situations in which `predicate_modifer` can be ignored as a part of the primary key..

Reply:
Trusting it now :P